### PR TITLE
Add timeout override

### PR DESCRIPTION
**Pull request by smneal**
### Context

Add the ability to set the timeout to a specified value.

### Change proposed in this pull request

Add a new option `-t` to override the default timeouts.

### Testing

Restored production backup to production AKS database with new parameter

`bin/konduit.sh -i apply_prod_2023-03-27.sql.gz -c -t 7200 apply-production -- psql`

### CODE REVIEW

File: scripts/konduit.sh [hunk 1]
```diff
@@ -15,10 +15,10 @@ help() {
    echo
 
    echo "Syntax:"
-   echo "   konduit [-a|-c|-h|-i file-name|-r redis-var] app-name -- command [args]"
+   echo "   konduit [-a|-c|-h|-i file-name|-r redis-var|-t 7200] app-name -- command [args]"
```
Comment (RMcVelia) [inline scripts/konduit.sh:18]: -t timeout

File: scripts/konduit.sh [hunk 2]
```diff
@@ -15,10 +15,10 @@ help() {
...
    echo "      Connect to the default database for app-name"
    echo
-   echo "or konduit [-a|-c|-h|-i file-name|-r redis-var] -d db-name -k key-vault app-name -- command [args]"
+   echo "or konduit [-a|-c|-h|-i file-name|-r redis-var|-t 7200] -d db-name -k key-vault app-name -- command [args]"
```
Comment (RMcVelia) [inline scripts/konduit.sh:21]: -t timeout

File: scripts/konduit.sh [hunk 3]
```diff
@@ -33,6 +33,7 @@ help() {
    echo "                     and contain a full connection URL."
    echo "   -r redis-var      Variable for redis cache [defaults to REDIS_URL if not set]"
    echo "                     Only valid for command redis-cli"
+   echo "   -t 3600           Timeout in seconds. Override default timeout."
```
Comment (RMcVelia) [inline scripts/konduit.sh:36]: -t timeout
maybe add default is 28800 but 3600 for psql backup/restore?

File: scripts/konduit.sh (hunk 1 above)
Comment (saliceti) [inline scripts/konduit.sh:18]: This should be descriptive, ie `-t timeout`

**Review by RMcVelia — APPROVED** [review]

**Pull request merged**